libs/sdk-core/src/binding.rs

```diff
@@ -83,13 +81,7 @@ pub fn node_info() -> Result<NodeState> {
 
 /// Cleanup node resources and stop the signer.
 pub fn disconnect() -> Result<()> {
-    block_on(async {
-        let shutdown_handler = BREEZ_SERVICES_SHUTDOWN.get();
-        match shutdown_handler {
-            None => Err(anyhow!("Background processing is not running")),
-            Some(s) => s.send(()).await.map_err(anyhow::Error::msg),
-        }
-    })
+    block_on(async { get_breez_services()?.disconnect().await })
```

Comment:
I think the `BREEZ_SERVICES_INSTANCE` has to be reset in order to be able to connect again later.

Reply:
I agree. OneCell only let you do a one time initialization so we might need to use a different mechanism here to hold the single breez services instance.

Reply:
The simplest thing I found that looks promising is
```rust
static BREEZ_SERVICES_INSTANCE: Lazy<Arc<Mutex<Option<Arc<BreezServices>>>>> =
    Lazy::new(|| Arc::new(Mutex::new(None)));
```

but its horrible.

We want it mutable, but also possibly empty. Wrapped in an `Arc` for sure. And holding a pointer (`Arc`) to the `BreezServices`, not the instance itself.

I'll keep looking, but if someone has a better idea, let me know.

Reply:
Why not this?

`static BREEZ_SERVICES_INSTANCE: Mutex<Option<Arc<BreezServices>>> = Mutex::new(None);`

Reply:
Good idea, tried it: 99f4aae90e87b93bb6917020122d8973dc07a98e

Had to wrap it in a `Lazy<>` though.

If this looks good, I'll do it for the other globals too.